wire-ios-request-strategy/Sources/E2EIdentity/E2EIRepository.swift

```diff
@@ -71,9 +71,11 @@ public final class E2EIRepository: E2EIRepositoryInterface {
         try await e2eiSetupService.registerTrustAnchor(trustAnchor)
     }
 
-    public func fetchFederationCertificate() async throws {
-        let federationCertificate = try await acmeApi.getFederationCertificate()
-        try await e2eiSetupService.registerFederationCertificate(federationCertificate)
+    public func fetchFederationCertificates() async throws {
+        let federationCertificates = try await acmeApi.getFederationCertificates()
+        for certificate in federationCertificates {
+            try await e2eiSetupService.registerFederationCertificate(certificate)
+        }
```

Comment:
if one fail, we fail all? it could be interesting to catch the one failing, no?

Reply:
Maybe we can find a way to report to the caller, like return an array of errors instead of throwing right away or pass a fail closure?

Reply:
Added a catch and logged the error, please check https://github.com/wireapp/wire-ios/pull/1120/commits/c3d1d3a6dcbb02ee4c98be0a37b8d237862ffa66